config-proxy/src/main/java/com/yahoo/vespa/config/proxy/filedistribution/FileDistributionRpcServer.java

```diff
@@ -77,7 +77,7 @@ private void declareMethods() {
     private static final int baseErrorCode = 0x10000;
     private static final int baseFileProviderErrorCode = baseErrorCode + 0x1000;
 
-    private static final int fileReferenceDoesNotExists = baseFileProviderErrorCode;
+    private static final int fileReferenceNotFound = baseFileProviderErrorCode;
```

Comment:
should this be capital letters, like ABORT, CONNECTION etc ?
I am not sure what is java practice.

Reply:
Both are used, but should probably be capital letters, yes. I want to do some more cleanup related to this, so will keep this in mind in a future PR